data/fields/climbing/rock.json

```diff
@@ -0,0 +1,11 @@
+{
+    "key": "climbing:rock",
+    "type": "combo",
+    "options": [
+        "limestone",
+        "sandstone",
+        "granite",
+        "basalt"
+    ],
```

Comment:
Well, my aim here was to make all the options translatable. I think this is important, because us non-native speakers usually know the rock types only in our language.

I removed `options` and added proper `strings.options` according to taginfo. Is it ok? Can we somehow suggest that people can add other options as well?